new-lamassu-admin/src/pages/OperatorInfo/TermsConditions.js

```diff
@@ -0,0 +1,166 @@
+import React, { useState } from 'react'
+import classnames from 'classnames'
+import * as R from 'ramda'
+import * as Yup from 'yup'
+import { Form, Formik, Field } from 'formik'
+import { gql } from 'apollo-boost'
+import { makeStyles } from '@material-ui/core'
+import { useQuery, useMutation } from '@apollo/react-hooks'
+
+import { Info2, Label2 } from 'src/components/typography'
+import { Switch } from 'src/components/inputs'
+import TextInputFormik from 'src/components/inputs/formik/TextInput'
+import { Button } from 'src/components/buttons'
+import ErrorMessage from 'src/components/ErrorMessage'
+
+import {
+  styles as globalStyles,
+  termsConditionsStyles
+} from './OperatorInfo.styles'
+
+const GET_CONFIG = gql`
+  {
+    config
+  }
+`
+
+const SAVE_CONFIG = gql`
+  mutation Save($config: JSONObject) {
+    saveConfig(config: $config)
+  }
+`
+
+const styles = R.merge(globalStyles, termsConditionsStyles)
+
+const useStyles = makeStyles(styles)
+
+const TermsConditions = () => {
+  const [showOnScreen, setShowOnScreen] = useState(false)
+  const [formData, setFormData] = useState(null)
+  const [error, setError] = useState(null)
+  const [saveConfig] = useMutation(SAVE_CONFIG, {
+    onCompleted: data => {
+      const { termsAndConditions } = data.saveConfig
+      setFormData(termsAndConditions)
+      setShowOnScreen(termsAndConditions.show)
+      setError(null)
+    },
+    onError: e => setError(e)
+  })
+
+  useQuery(GET_CONFIG, {
+    onCompleted: data => {
+      const { termsAndConditions } = data.config
+      setFormData(termsAndConditions ?? {})
+      setShowOnScreen(termsAndConditions?.show ?? false)
+    }
+  })
+
+  const save = it => {
+    setError(null)
+    return saveConfig({
+      variables: { config: { termsAndConditions: it } }
+    })
+  }
+
+  const handleEnable = () => {
+    const s = !showOnScreen
+    save({ show: s })
+  }
+
+  const classes = useStyles()
```

Comment:
Move this to the top of the Component. Generally hooks should be the first thing called within the component function.